Index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>landing Page</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="nav__leftside">
+                <div class="nav__logo">Medical Core
+                </div>
+                <div class="nav-lists">
+                    <div class="nav-item">Home</div>
+                    <div class="nav-item">Product</div>
+                    <div class="nav-item">Pricing</div>
+                    <div class="nav-item">Contact</div>
+                
+                </div>
+            </div>
+            <div class="nav__rightside">
+                <button class="button button--link">Login</button>
+                <button class="button">join us</button>
+            </div>
+        </nav>
+    <section class="middle__section">
+        <div class="middle__sectiontitle">Welcome
+        </div>
+        <div class="middle__sectiondescr">A Great Place to Receive Care</div>
+        <div class="middle__sectionsubt">Get your best looking smile now!</div>
+        <div class="middle__sectionbtn">
+            <button class="button">Join Us</button>
+            <button class="button button--set">learn more</button>
+        </div>
+    </section>
+</header>
+<main class="main">
+<div class="intro">
+    <img class="main__introimg" src="./images/thumb-concept.png" alt="" >
+<div class="main__descr">
+    <div class="main__border"></div>
+    <div class="main__title">Meet Our Experts</div>
+    <div class="main__text">Problems trying to resolve the conflict between 
+        the two major realms of Classical physics: 
+        Newtonian mechanics
+    </div>
+    <div class="main__link">Learn More</div>
+</div>
+</div>
+<div class="team-layout">
+    <div class="team__title">Team</div>
```

Comment:
Here the block is team-layout and it's elements must be team-layout__title, team-layout__(...).
 every element within this block must start by team-layout as it is a block name